### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anamika1818**

### CODE REVIEW

**Comment by WyriHaximus** [comment]
TL;DR This PR is rejected because it's classified as spam.

The license will not be changed like this. While I approciate that you took the time to create a PR on one of my projects, it seems like you create this PR based on a youtube video discussed on this post [`How One Guy Ruined #Hacktoberfest2020 #Drama`](https://joel.net/how-one-guy-ruined-hacktoberfest2020-drama).
[Which is causing a shitstorm](https://twitter.com/WyriHaximus/status/1311614161529511938) [on Twitter at the moment](https://twitter.com/jmikola/status/1311988470156460033) [about the metric ton](https://twitter.com/BauerVadim/status/1311939074635296769) [of spam it creates](https://twitter.com/wesbos/status/1311674713962418180).

Whilest I detest creating PR's like this, I'd gladly help you creating a meaningfull PR. There are a bunch of open issues specially created for #Hacktoberfest on my repositories I suggest you take a look at: [`Hacktoberfest Issues`](https://github.com/search?q=user%3AWyriHaximus+label%3Ahacktoberfest+is%3Aopen)

So if you want, please reach out, I gladly help you get started and make a positive impact.
